src/controller/CHIPDeviceController.cpp

```diff
@@ -619,6 +619,15 @@ CHIP_ERROR DeviceCommissioner::Init(CommissionerInitParams params)
 
     params.systemState->SessionMgr()->RegisterRecoveryDelegate(*this);
 
+#if 0 // TODO: SessionID persistence results in bugs when instantiating multiple DeviceCommissioners
+      // on multiple fabrics due to not doing persistence completely, i.e not persisting the session ID
+      // upon completion of CASE. This results in the 2nd-to-N commissioners upon initialization picking
+      // up a stale value for the next available session ID and colliding with a previously picked local session ID
+      // from another commissioner instance.
+      //
+      // For now, just disable all persistence.
+      //
```

Comment:
This comment is misleading to me.  It's not the persistence that is really root cause.  It's that we have multiple session ID spaces being used simultaneously within a single session manager that expects and requires each session ID to be unique.

Reply:
OK, I think I understand.  This is behavior that will occur now with the fix / workaround that makes the allocator internals static.

However, this seems to imply that someday we will have persistence managed here within each DeviceCommissioner instance.  But we know now that management of session ID allocation (and, if necessary, persistence) needs to be centralized within and controlled by the session manager.

Can we maybe improve this comment then, or even remove it then?  Whomever ultimately fixes these problems will not gain much useful insight from this.  If there is a TODO here at all, it should be something to the effect: "We cannot reinstantiate session ID allocator state from each fabric-scoped commissioner individually because the session ID allocator space is and must be shared for all users of the Session Manager.  Disable persistence for now."

Reply:
You're right, I was explaining more the current state of affairs, and less where it should be going. Your comment is better suited here.